src/health-check.js

```diff
@@ -0,0 +1,61 @@
+const { createServer } = require('http')
+const { logger, serializeError } = require('./logging')
+const { getPeerId } = require('./peer-id')
+const { defaultDispatcher, searchCarInDynamo } = require('./storage')
+const { blocksTable, primaryKeys } = require('./config')
+
+class HealthCheck {
+  constructor() {
+    this.logger = logger
```

Comment:
How come this is set as a instance property? Why not just reference `logger` from the `logging.js` module? In `checkReadiness` you call `logger(...)` but in `startServer` you call `this.logger(...)`. Some consistency either way 🙂🙏.

Reply:
Done!